src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyPscCountValidation.java

```diff
@@ -15,13 +15,20 @@
 
     private final PscService pscService;
 
-    public CompanyPscCountValidation(PscService pscService) {
+    private final boolean companyPscCountValidationFeatureFlag;
+
+    public CompanyPscCountValidation(PscService pscService, boolean companyPscCountValidationFeatureFlag) {
         this.pscService = pscService;
+        this.companyPscCountValidationFeatureFlag = companyPscCountValidationFeatureFlag;
     }
 
     @Override
     public void validate(CompanyProfileApi profileToValidate) throws EligibilityException, ServiceException {
         LOGGER.info("Validating Company PSCs for: {}", profileToValidate.getCompanyNumber());
+        if (!companyPscCountValidationFeatureFlag) {
```

Comment:
I'd move the if feature flag check to first line of method, might be confusing if log on fist line prints then says it is off.

Reply:
actually, no, it's ok as is